modules/programs/fish.nix

```diff
@@ -253,6 +253,15 @@ in {
         '';
       };
 
+      generateCompletions = mkOption {
+        type = types.bool;
+        default = true;
+        defaultText = literalExpression "true";
+        description = ''
+          Enable or disable the automatic generation of completions based upon installed man pages.
+        '';
+      };
```

Comment:
```suggestion
      generateCompletions = mkEnableOption "the automatic generation of completions based upon installed man pages" // {
        default = true;
      };
```

Reply:
Thanks for the feedback! I've updated the patch.